src/display/font_loader.js

```diff
@@ -393,8 +393,8 @@ class FontFaceObject {
     return rule;
   }
 
-  getPathGenerator(objs, character) {
-    if (this.compiledGlyphs[character] !== undefined) {
+  tryGetPathGenerator(objs, character) {
+    if (this.compiledGlyphs.hasOwnProperty(character)) {
```

Comment:
`hasOwnProperty` checks tend to be more expensive, and `paintChar` where this is called seems to be called a lot, so let's try to avoid using that.

Reply:
Sure. I reverted to the old way, but I think it is less clear than using `hasOwnProperty`. I believe performance-wise it doesn't matter here which is used https://www.measurethat.net/Benchmarks/Show/1303/0/objecthasownproperty-vs-object-in-vs-object-vs-arrayind#latest_results_block